### PR TITLE
Change to latest project setup

### PR DESCRIPTION
**Pull request by wmontwe**
### :tophat: What is the goal?

Streamline project configuration

### :unicorn: How is it being implemented?

Bump Gradle 6.9 -> 6.9.1

Fix versioning script to be used in git submodule project

### :thinking: DOD Checklist

- [x] My code follows the code style and naming conventions of this project.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have updated the documentation.
- [ ] I have updated the changelog accordingly.

### CODE REVIEW

File: gradlePlugin/securestore-dependency/src/main/kotlin/care/data4life/sdk/securestore/dependency/Version.kt [hunk 1]
```diff
@@ -62,13 +62,13 @@ object Version {
             /**
              * [hc-util-kmp](https://github.com/d4l-data4life/hc-util-kmp)
              */
-            const val utilSdk = "1.7.0"
+            const val utilSdk = "1.8.0"
```
Comment (bitPogo) [inline gradlePlugin/securestore-dependency/src/main/kotlin/care/data4life/sdk/securestore/dependency/Version.kt]: Should this not go to 1.10.0?

File: gradlePlugin/securestore-dependency/src/main/kotlin/care/data4life/sdk/securestore/dependency/Repository.kt [hunk 2]
```diff
@@ -38,9 +39,11 @@ object Repository {
         // Maven dependency group
         "care.data4life.hc-sdk-kmp",
         "care.data4life.hc-util-sdk-kmp",
+        "care.data4life.hc-util-test-sdk-kmp",
         "care.data4life.hc-fhir-sdk-kmp",
         "care.data4life.hc-fhir-helper-sdk-kmp",
-        "care.data4life.d4l-data-donation-sdk-kmp"
+        "care.data4life.d4l-data-donation-sdk-kmp",
```
Comment (bitPogo) [inline gradlePlugin/securestore-dependency/src/main/kotlin/care/data4life/sdk/securestore/dependency/Repository.kt:45]: Why we need DD here?

Comment (wmontwe) [reply gradlePlugin/securestore-dependency/src/main/kotlin/care/data4life/sdk/securestore/dependency/Repository.kt:45]: that's the default config

Comment (bitPogo) [reply gradlePlugin/securestore-dependency/src/main/kotlin/care/data4life/sdk/securestore/dependency/Repository.kt:45]: oke

File: gradlePlugin/securestore-dependency/src/main/kotlin/care/data4life/sdk/securestore/dependency/Version.kt (hunk 1 above)
Comment (wmontwe) [reply gradlePlugin/securestore-dependency/src/main/kotlin/care/data4life/sdk/securestore/dependency/Version.kt]: Well, now yes

**Review by bitPogo — APPROVED** [review]

**Pull request merged**